apps/designer/app/designer/features/props-panel/control.tsx

```diff
@@ -142,84 +220,150 @@ const RangeControl = ({
     </Flex>
   </Flex>
 );
+*/
 
 const NotImplemented = () => <div />;
 
-type PrimitiveControlProps = BaseControlProps & {
-  type: "array" | "boolean" | "date" | "number" | "object" | "text";
+const includes = <T extends string>(arr: readonly T[], v: string): v is T => {
```

Comment:
I think we need this as a reusable utility somewhere ... is there a good package that already has it or can we put it somewhere else?